app/models/user.rb

```diff
@@ -4,4 +4,12 @@ class User < ActiveRecord::Base
 
 	validates :name, presence: true
```

Comment:
I would vote adding an email field that is required to be unique.
On Feb 7, 2015 12:07 PM, "Kara A Carrell" <notifications@github.com> wrote:

> In app/models/user.rb
> <https://github.com/chi-squirrels-2015/flippit-flashcards/pull/5#discussion_r24291305>
> :
>
> > @@ -4,4 +4,12 @@ class User < ActiveRecord::Base
> >
> >  	validates :name, presence: true
>
> should this not also be unique? how are we making sure users are unique?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chi-squirrels-2015/flippit-flashcards/pull/5/files#r24291305>
> .
>